component.yaml

```
@@ -0,0 +1,18 @@
+platform: iOS
+buildPlatform:
+displayName: Tuist Commands
+description: "Run Tuist Commands for Project."
+inputs:
+- key: "AC_TUIST_PATH"
+  defaultValue: "$AC_REPOSITORY_DIR"
+  isRequired: false
+  title: Project Path
+  description: The path to the directory that contains the definition of the project. Defaults to current directory.
```

I did not understand what do you mean by "definition of the project"

---

But for second sentence I think following option is more undertanable. Is't it?
```suggestion
  description: The path to the directory that contains the definition of the project. By default, it is set to the root directory of the cloned repository.
```